src/components/CardItem.tsx

```diff
@@ -0,0 +1,40 @@
+import { ButtonLink } from "./ButtonLink";
+import { ROUTES } from "../routes";
+
+interface Props {
+	imgCard: string;
```

Comment:
to bych dal bez toho prizviska Card, jen img, title a description nebo paragraph